io/src/obj_io.cpp

```diff
@@ -382,6 +380,7 @@ pcl::OBJReader::readHeader (const std::string &file_name, pcl::PCLPointCloud2 &c
         continue;
 
       // Trim the line
+      //TOOD: we can easily do this without boost
```

Comment:
This is just a `find_first_not_of` with `isspace` character set

Reply:
That is a valid point but here it is not. We do the following steps:
`getline` -> `trim` -> check if line begins with `{#, v, vt, vn, mtllib}`

Reply:
You're right, and the `boost::split`, now `pcl::split`, can handle whitespace at the end, correct?

Reply:
yes, `pcl::split` can handle whitespace at the end. I can do it in some other PR. To keep things clean and easy to review